doc/source/theory/Mellin.rst

```diff
@@ -130,10 +130,10 @@ the Mellin inverse.
 - for the sums :math:`S_{-m}(N)` and :math:`m > 0` we use:
 
   .. math ::
-    S_{-m}(N) = \frac{\eta}{2} (S_{m}(N / 2) - S_{m}((N - 1) / 2)) - c_{m}
+    S_{-m}(N) = \frac{\eta}{2} (S_{m}(N / 2) - S_{m}((N - 1) / 2)) - d_{m}
 
   where formally :math:`\eta = (-1)^N` but in all singlet-like quantities it has to be analytically continued with 1
-  and with -1 elsewise and :math:`c_{m}= \left [ log(2), 1/2 \zeta_{2}, 3/4 \zeta_{3}, 7/8 \zeta_{4}, 15/16 \zeta_{5} \right]`
+  and with -1 elsewise and :math:`d_{m}= \left [ \log(2), 1/2 \zeta_{2}, 3/4 \zeta_{3}, 7/8 \zeta_{4}, 15/16 \zeta_{5}, \ldots \right]`
```

Comment:
@giacomomagni this definition does not match with [MuselliPhD, Eq. (B.5.4d)] (zeta is evaluated at -k there); can you please confirm (maybe you have a second source or just a simple example, in case Muselli is wrong)? are we using this relation somewhere in the code?

Reply:
So this relation should be equivalent to eq 5.4d

Reply:
I think I actually used: [eq.510](https://arxiv.org/pdf/0901.3106.pdf)

Reply:
I guess then we trust in your reference ...

Reply:
I added a test to check the consistency of these 2 formulas: 

https://github.com/N3PDF/eko/blob/9606618337c08df4e2118ce3c10be0862b1562c6/tests/eko/test_harmonics.py#L127